assets/src/components/editing/elements/utils.tsx

```diff
@@ -19,8 +19,10 @@ export function updateModel<T extends ContentModel.ModelElement>(
 }
 
 export const onEditModel = <T extends ContentModel.ModelElement>(model: T) => {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
onEditModel should probably be named useEditModel since it's acting as a custom hook, but I chose to just ignore this error for now.